.github/workflows/test.yml

```diff
@@ -456,12 +456,16 @@ jobs:
         fuzz-seconds: 300
         dry-run: false
         language: go
+    - name: Set artifacts_path in env (workaround for actions/upload-artifact#176)
```

Comment:
This was a (seemingly) undocumented change we discovered while doing a similar bump in https://github.com/tailscale/go/pull/100/files